json/src/main/scala/fs2/data/json/package.scala

```diff
@@ -23,6 +23,7 @@ import json.internals._
 
 import cats._
 import fs2.data.text.render.Renderable
+import fs2.data.json.Token.StartArray
```

Comment:
Minor: Let's align on either importing the concrete tokens or using `Token.Xyz` – current `StartArray` and `EndArray` are handled differently.